sway-lib-core/src/never.sw

```diff
@@ -2,6 +2,25 @@ library never;
 
 use ::ops::{Not, Eq, Ord};
 
+/// `Never` is assigned to a block that is deterministically aborted. This can be
```

Comment:
I was hoping we'd copy the documentation from https://doc.rust-lang.org/std/primitive.never.html verbatim (to the extent possible). That's what we've tried to do in other spots in the standard library. Is that reasonable here or are there some major differences between our `Never` type and Rust's `Never` type?

Reply:
Copied the link above to the possible extend. Rust has more use cases to
the Never type than Sway currently has, so the sections were not included in the
docstring as they are not applicable to the Sway Never type.